controllers/constants/constants.go

```diff
@@ -49,8 +49,14 @@ const (
 	KServeCACertConfigMapName  = "odh-kserve-custom-ca-bundle"
 	ODHGlobalCertConfigMapName = "odh-trusted-ca-bundle"
 	ODHCustomCACertFileName    = "odh-ca-bundle.crt"
+	KServeGatewayName          = "kserve-local-gateway"
```

Comment:
"I don't think we should use 'envvar' because it is specifically for the KServe local gateway. If the name changes, then we can update this, but I don't think there is a possibility of it changing.

Reply:
Well... if the users do the manual install, there is the chance they can use any name.
But, this is just a recommendation... definitely not a blocking comment.

Reply:
let's leave it for now.